### PR TITLE
Update top level README to reflect consolidation of vault and vault signature provider.

### PR DESCRIPTION
**Pull request by opi-smccoole**

### CODE REVIEW

**Review by mjohnson12 — APPROVED** [review]
👍

File: README.md [hunk 1]
```diff
@@ -5,15 +5,25 @@
...
+EOSIO SDK for Swift: Vault consists of two main components; Vault and Vault Signature Provider.  
 
-The Vault library is a required dependency of the [EOSIO SDK for Swift: Vault Signature Provider](https://github.com/EOSIO/eosio-swift-vault-signature-provider). It additionally provides key generation, management and signing functions that can be called directly.
+Vault is a utility library for working with public/private keys and signing with Apple's Keychain and Secure Enclave.
+
+Vault Signature Provider is a pluggable signature provider for [EOSIO SDK for Swift](https://github.com/EOSIO/eosio-swift). It allows for signing transactions using keys stored in Keychain or the device's Secure Enclave.
+
+The Vault component is a required dependency of Vault Signature Provider. It additionally provides key generation, management and signing functions that can be called directly.
 *All product and company names are trademarks™ or registered® trademarks of their respective holders. Use of them does not imply any affiliation with or endorsement by them.*
```
Comment (brandonfancher) [inline README.md:15]: Can we add a line break above the italicized text here? When rendered, it's rendering as part of the same paragraph of the sentence that precedes it.

File: README.md [hunk 2]
```diff
@@ -22,28 +32,147 @@ The Vault library is a required dependency of the [EOSIO SDK for Swift: Vault Si
...
+All signature providers must conform to the [EosioSignatureProviderProtocol](https://github.com/EOSIO/eosio-swift/blob/master/Sources/EosioSwift/EosioSignatureProviderProtocol/EosioSignatureProviderProtocol.swift) Protocol.
+
 ## Prerequisites
 
-* Xcode 10 or higher
-* CocoaPods 1.5.3 or higher
-* For iOS, iOS 11.3+
+* Xcode 11 or higher
+* CocoaPods 1.9.3 or higher
+* Swift Package Manager swift-tools 5.2 or higher
```
Comment (brandonfancher) [inline README.md:50]: Is "swift-tools..." supposed to be a new bullet point, or same as is here?

File: README.md [hunk 3]
```diff
@@ -22,28 +32,147 @@ The Vault library is a required dependency of the [EOSIO SDK for Swift: Vault Si
...
+* `EosioVaultSignatureProvider.signTransaction(request:completion:)` signs an [`EosioTransaction`](https://github.com/EOSIO/eosio-swift/blob/master/Sources/EosioSwift/EosioTransaction/EosioTransaction.swift).
+* `EosioVaultSignatureProvider.getAvailableKeys(...)` returns an array containing the public keys associated with the private keys that the object is initialized with.
+
+To initialize the implementation:
+
+* `EosioVaultSignatureProvider.init(accessGroup:requireBio:)` initializes the signature provider.
+  * `accessGroup`: [Learn more about Access Groups here.](https://developer.apple.com/documentation/security/keychain_services/keychain_items/sharing_access_to_keychain_items_among_a_collection_of_apps)
+  * `requireBio`: Defaults to `false`. Some keys may require biometric authentication no matter what this flag is set to. For keys that do not require biometric authentication themselves, this flag can force the bio check.
+
+Other Keychain and/or Secure Enclave functionality can be accessed by calling methods directly on [EOSIO SDK for Swift: Vault](https://github.com/EOSIO/eosio-swift-vault), which is included with this library as a dependency.
```
Comment (brandonfancher) [inline README.md:170]: This line might need massaging. It appears to still be written assuming vault is in a different repo.

**Review by brandonfancher — APPROVED** [review]
Approved with minor edits requested. Nicely done. 👍

File: README.md (hunk 2 above)
Comment (opi-smccoole) [reply README.md:50]: Same as here.  swift-tools is the main versioned component of Swift Package Manager.  Maybe parentheses or leave of the swift-tools portion and just say 5.2 or higher?

Comment (opi-smccoole) [reply README.md:50]: I think I will just drop "swift-tools" out of this line and say "Swift Package Manager 5.2 or higher"

File: README.md (hunk 3 above)
Comment (opi-smccoole) [reply README.md:170]: Oops!  Yeah missed that one.  Updated to:

Other Keychain and/or Secure Enclave functionality can be accessed by calling methods directly on the Vault component, which is included with Vault Signature Provider as a dependency.

File: README.md (hunk 1 above)
Comment (opi-smccoole) [reply README.md:16]: Done!

**Pull request merged**